scripts/common_variables

```diff
@@ -48,7 +48,7 @@ EXTRA_QEMU_ARGS=${EXTRA_QEMU_ARGS:-""}
 EXTRA_CPU=",v=on,vlen=256,elen=64"
 
 CPU_TYPE=rv64${EXTRA_CPU}
-CPU_ARGS="${CPU_TYPE},smaia=true,ssaia=true,sscofpmf=true"
+CPU_ARGS="${CPU_TYPE},smaia=true,ssaia=true,sscofpmf=true,smstateen=true"
```

Comment:
Did you keep this patch intentionally to preserve the required history of required extensions ? 
If that's the purpose, it is fine. Otherwise, we can squash it into the next patch where we are changing it to max cpu.

Reply:
It is intentional. the max change being independently revertable seems like a good idea to me...